src/components/header/Header.jsx

```diff
@@ -18,7 +18,12 @@ const Header = () => {
           <ButtonPill text="Log in" onClick={logInOnClick} />
         </div>
       ) : (
-        <p className="ml-auto content-end text-white">I'm logged in!!!!</p>
+        <img
+          className="m1-auto content-end rounded-full"
+          src={rootStore.userStore.userProfileImageUrl}
+          alt="" //I added this bcuz of the "rule" in esling? I got a caution/error prompt
```

Comment:
nice good find! the `alt` property is used for screen readers for folks with visual impairments (i think), or when the picture does not load properly and users will still have a way of identifying what the image was supposed to be